llvm/test/MC/RISCV/corev/hwlp/fixups.s

```diff
@@ -0,0 +1,14 @@
+# RUN: llvm-mc -triple riscv32 --mattr=+xcorevhwlp -show-encoding %s \
+# RUN:     | FileCheck -check-prefix=CHECK-FIXUP %s
+
+.LBB0:
+cv.starti 0, .LBB1
+# CHECK-FIXUP: fixup A - offset: 0, value: .LBB1, kind: fixup_riscv_cvpcrel_ui12
+cv.endi 0, .LBB1
+# CHECK-FIXUP: fixup A - offset: 0, value: .LBB1, kind: fixup_riscv_cvpcrel_ui12
+cv.setup 0, zero, .LBB1
+# CHECK-FIXUP: fixup A - offset: 0, value: .LBB1, kind: fixup_riscv_cvpcrel_ui12
+cv.setupi 0, .LBB1, 0
+# CHECK-FIXUP: fixup A - offset: 0, value: .LBB1, kind: fixup_riscv_cvpcrel_urs1
```

Comment:
Are those tests enough to test fixups/relocations?

Reply:
I would add a relocation check too. Emit an object and disassemble it, showing relocations and check those work. This then becomes a good test too of fixups->relocs.

Reply:
I added them in 77c7649. This is inspired by https://github.com/openhwgroup/corev-llvm-project/blob/b89420ac28e2ae99e4d0fc45e59c2a197ec7aa2b/llvm/test/MC/RISCV/relocations.s#L169-L172